### DISCUSSION TITLE
Changes to how this is functioning

### DISCUSSION THREAD

**Discussion by ExperiencersInternational** [ExperiencersInternational/tvsetup#4] [Announcements]
Hi there, some changes to how this works due to the recent crackdown:

- Certain streams are being taken down just to be cautious, however if you necessarily need it, use an older release version

We'll be using a new EPG as well, I don't know what I'll have to do to get this working though. This may also bypass the need for self-hosting EPG for channels which don't have them.

**Comment by coldjim** [comment]
Hi, I am testing this with VLC but I can only get the BBC channels to work? I have used the older releases as suggested but ITV, Channel 4 etc don't seem to work. Is there a way to fix this? Ideally I'd like to add this to the TVs in my house since I don't have aerial. Any help would be appreciated.

**Comment by ExperiencersInternational** [comment]
Please see #11 @coldjim 

Some channels may be back now idk
